docs/next/components/mdx/MDXComponents.tsx

```diff
@@ -327,23 +334,24 @@ export default {
      */
     const { version } = useVersion();
     const { src } = props;
-    if (version === "master" || !src.startsWith("/images/")) {
+    if (!src.startsWith("/images/")) {
       return (
-      <Zoom wrapElement="span" wrapStyle={{display: 'block'}}>
         <span className="block mx-auto">
           <Image {...(props as any)} />
         </span>
-      </Zoom>
       );
     }
 
-    const resolvedPath = new URL(
-      path.join("versioned_images", version, src.replace("/images/", "")),
-      "https://dagster-docs-versioned-content.s3.us-west-1.amazonaws.com"
-    ).href;
+    const resolvedPath =
+      version === "master"
+        ? src
+        : new URL(
+            path.join("versioned_images", version, src.replace("/images/", "")),
+            "https://dagster-docs-versioned-content.s3.us-west-1.amazonaws.com"
+          ).href;
 
     return (
-      <Zoom wrapElement="span" wrapStyle={{display: 'block'}}>
+      <Zoom wrapElement="span" wrapStyle={{ display: "block" }}>
```

Comment:
actual fix here

above changes are just autoformatting as i went through this file